src/storage/src/source/persist_source.rs

```diff
@@ -246,8 +246,8 @@ where
 
     let mut fetcher_builder = OperatorBuilder::new(
         format!(
-            "persist_source {:?}: part fetcher {}",
```

Comment:
Please don't give operators worker-specific names. Even if it works, it's not defined and could cause surprised down the road.

Reply:
👌 good catch!

Reply:
@antiguru do docs on operator name conventions exist anywhere? I've looked for them before for stuff like this and not found them, so if there are, maybe we need to link to them from more places

Reply:
side note: operator name are all over the place :see_no_evil:

Reply:
Nope, these docs don't exist! In Materialize, I've tried to use descriptive names that are the same on all workers, because we can get the per-worker data anyways.